spec/lib/support_ticket_creator_spec.rb

```diff
@@ -33,6 +33,20 @@
 
       expect { SupportTicketCreator.call(messy_args) }.to_not raise_exception
     end
+
+    it "Raises an error when 422 is returned" do
+      stub_support_api_invalid_raise_support_ticket(support_ticket.payload)
+
+      expect { SupportTicketCreator.call(args) }.to raise_exception(GdsApi::HTTPUnprocessableEntity)
+    end
+
+    it "Doesn't raises an error when 422 is returned because of suspended user" do
```

Comment:
Minor: this test name should be tweaked a bit so that reads well with the `it`.

Perhaps: `it "doesn't raise an error when a 422 error is returned for a suspended user"`

Reply:
Fixed!